README.md

```diff
@@ -18,22 +18,18 @@ This document explains the Classification Extension to the [SpatioTemporal Asset
 
 | Field Name              | Type                | Description |
 | ----------------------- | ------------------- | ----------- |
-| classification:classed  | `Classed`           | **REQUIRED**. Classes in the dataset |
+| classification:class_list  | `[string]`           | **REQUIRED**. Summary of class names in the dataset |
+
+| Field Name              | Type                | Description |
+| ----------------------- | ------------------- | ----------- |
+| classification:classes  | `[Class]`         | **REQUIRED**. Categorized classes in the dataset |
 
 | Field Name               | Type               | Description |
 | ------------------------ | ------------------ | ----------- |
-| classification:bitmask   | `[Bitmask]`        | **REQUIRED**. Classes in the dataset |
-
-### Classed Object
-
-*Describes multiple classes*
+| classification:bitmask   | `[Bitmask_Part]`        | **REQUIRED**. Classes stored in bit ranges in the dataset |
```

Comment:
Should they really all be required? I think I have use cases in mind where I don't need bitmasks.

Reply:
errr, no, that doesn't make sense